modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/checkpoint/IgniteMassLoadSandboxTest.java

```diff
@@ -297,6 +301,64 @@ public void testDataStreamerContinuousPutMultithreaded() throws Exception {
         }
     }
 
+    /**
+     * Test that WAL segments that are fully covered by checkpoint are logged
+     * @throws Exception if failed.
+     */
+    public void testClearedWalsLogged() throws Exception {
+        try {
+            final GridStringLogger log0 = new GridStringLogger();
+
+            final IgniteConfiguration cfg = getConfiguration("testClearedWalsLogged");
+
+            cfg.setGridLogger(log0);
+
+            cfg.getDataStorageConfiguration().setWalAutoArchiveAfterInactivity(10);
+
+            final Ignite ignite = G.start(cfg);
+
+            ignite.cluster().active(true);
+
+            final IgniteCache<Object, Object> cache = ignite.cache(CACHE_NAME);
+
+            cache.put(1, new byte[1024]);
+
+            forceCheckpoint();
+
+            Thread.sleep(200); // needed by GridStringLogger
+
+            final String log = log0.toString();
+
+            System.out.println(log);
```

Comment:
Looks like a leftover from test development, please remove.

Reply:
This if for keeping visual control if smth goes wrong. Do we need to delete it?